classes/checkout/CheckoutDeliveryStep.php

```diff
@@ -120,6 +121,18 @@ public function getGiftCostForLabel()
         return '';
     }
 
+    public function setSeparatePackagesAllowed($separatePackagesAllowed)
```

Comment:
Hi, I think it could be useful to add phpDoc 😊  ?

I don't know if we can type arguments and return type 🤔  wdyt @Progi1984 ? should we introduce strong type hint even if rest of the class does not use it ?